app/src/test/java/org/simple/clinic/newentry/PatientEntryUpdateTest.kt

```diff
@@ -241,4 +241,54 @@ class PatientEntryUpdateTest {
         )
   }
 
+  @Test
```

Comment:
Can we reword the commit message to better reflect that we've done more than just add the district field error validation test?

Ex: `Add test for address fields validation checks`

Reply:
I have added a commit for each test that's why I wrote specific error names. the address field will have to combine 3 different tests and commits. Is that what you want me to do here? @vinaysshenoy

Reply:
If you look at the contents of this commit, it adds tests for both the `Colony/Village` field as well as the `District` field. The commit message just mentions that the test for the `District` field was added.

My intention is to make the commit message and the commit contents make sense when taken together.

Reply:
Ok, resolved